pkgs/development/python-modules/readme/default.nix

```diff
@@ -20,15 +20,15 @@ buildPythonPackage rec {
   ];
 
   propagatedBuildInputs = [
-    readme_renderer
+    readme-renderer
   ];
 
   checkPhase = ''
     pytest
   '';
 
   # tests are not included with pypi release
-  # package is not readme_renderer
+  # package is not readme-renderer
```

Comment:
Not your fault, but I don't understand the old (or new) version of this comment.